src/index.html

```diff
@@ -15,8 +15,110 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,
+      300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,
+      900&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
This line of code is too long. It's a good practice to keep lines of code to 80 characters max. This way, it is just easier to read.